Package doesn't appear to work under .NET Core or .NET 5

I've created a new `netcoreapp3.1` WinForms app and added a package reference to the latest version of the package.

```xml
<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>netcoreapp3.1</TargetFramework>
    <UseWindowsForms>true</UseWindowsForms>
    <Prefer32Bit>true</Prefer32Bit>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="unvell.D2DLib" Version="1.2.2" />
  </ItemGroup>

</Project>
```

NuGet restore completes succesfully. It doesn't appear to have added any managed assembly references to the compilation, as I see error: 

> error CS0246: The type or namespace name 'unvell' could not be found (are you missing a using directive or an assembly reference?)

Looking in the `obj/project.assets.json` file I see:

```json
      "unvell.D2DLib/1.2.2": {
        "type": "package",
        "build": {
          "build/unvell.D2DLib.targets": {}
        }
      },
```

The `.targets` file is picked up. I see the linked `d2dlib32.dll` in the project tree.

![image](https://user-images.githubusercontent.com/350947/101900346-70cd4000-3c03-11eb-819c-5cea170a381a.png)

If I change `<TargetFramework>` to `net4.8`, the compilation error goes away. The assets file now shows:

```json
      "unvell.D2DLib/1.2.2": {
        "type": "package",
        "compile": {
          "lib/net20/d2dlibexport.dll": {},
          "lib/net20/d2dwinform.dll": {}
        },
        "runtime": {
          "lib/net20/d2dlibexport.dll": {},
          "lib/net20/d2dwinform.dll": {}
        },
        "build": {
          "build/unvell.D2DLib.targets": {}
        }
      },
```

For some reason, NuGet is excluding the `compile` and `runtime` items when targeting `netcoreapp3.1` and `net5.0-windows`. My guess is the `.nuspec` file is at fault.

--- a/package/unvell.D2DLib.nuspec
+++ b/package/unvell.D2DLib.nuspec
@@ -15,7 +15,7 @@
     <description>Provides the hardware-accelerated high performance immediate mode rendering functionality via Direct2D API.</description>
     <summary>Provides the hardware-accelerated high performance immediate mode rendering functionality via Direct2D API.</summary>
     <copyright>Copyright (c) 2012-2019 Jingwood, unvell.com, all rights reserved.</copyright>
-    <tags>direct2d hardward-acceleration drawing draw custom-draw rendering paint gpu performance immediate bitmap memory-bitmap direct2d-api graphics-context</tags>
+    <tags>direct2d hardware-acceleration drawing draw custom-draw rendering paint gpu performance immediate bitmap memory-bitmap direct2d-api graphics-context</tags>
     <dependencies>
       <group targetFramework="net20">
       </group>

--- a/package/unvell.D2DLib.nuspec
+++ b/package/unvell.D2DLib.nuspec
@@ -17,18 +17,16 @@
     <copyright>Copyright (c) 2012-2019 Jingwood, unvell.com, all rights reserved.</copyright>
     <tags>direct2d hardware-acceleration drawing draw custom-draw rendering paint gpu performance immediate bitmap memory-bitmap direct2d-api graphics-context</tags>
     <dependencies>
-      <group targetFramework="net20">
-      </group>
-      <group targetFramework="netcore">
-      </group>
+      <group targetFramework="net20" />
+      <group targetFramework="netstandard1.0" />
     </dependencies>
   </metadata>
   <files>
     <file src="logo_d2dlib.png" target="" />
     <file src="..\binary\x86-build\d2dlibexport.dll" target="lib\net20" />
     <file src="..\binary\x86-build\d2dwinform.dll" target="lib\net20" />
-    <file src="..\binary\x86-build\d2dlibexport.dll" target="lib\netcore" />
-    <file src="..\binary\x86-build\d2dwinform.dll" target="lib\netcore" />
+    <file src="..\binary\x86-build\d2dlibexport.dll" target="lib\netstandard1.0" />
+    <file src="..\binary\x86-build\d2dwinform.dll" target="lib\netstandard1.0" />
     <file src="..\binary\x86-build\d2dlib32.dll" target="build" />
     <file src="unvell.D2DLib.targets" target="build" />
     <file src="..\README.md" target="" />

